src/PharoLauncher-CLI-Tests/PhLTemplateInfoCliCommandTest.class.st

```diff
@@ -0,0 +1,60 @@
+Class {
+	#name : #PhLTemplateInfoCliCommandTest,
+	#superclass : #PhLTemplatesCliCommandTest,
+	#category : #'PharoLauncher-CLI-Tests'
+}
+
+{ #category : #tests }
+PhLTemplateInfoCliCommandTest >> testTemplateInfoShouldSucceed [
+	self createOfficialRepositoryInFileSystemWithTemplatesInIt. 
```

Comment:
rename #createOfficialRepositoryInFileSystemWithTemplatesInIt to #createOfficialTemplatesCategoryWithFakeTemplate ?